axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -186,6 +189,21 @@ public Builder tagsConfiguration(TagsConfiguration tagsConfiguration) {
             return this;
         }
 
+        /**
+         * Registers the given {@code channelCustomization}, which configures the underling {@link ManagedChannelBuilder} used
+         * to set up connections to AxonServer.
+         * <p>
+         * This method may be used in case none of the operations on this Builder provide support for the required
+         * feature.
+         *
+         * @param channelCustomization A function defining the customization to make on the ManagedChannelBuilder
+         * @return this builder for further configuration
+         */
+        public Builder channelCustomizer(UnaryOperator<ManagedChannelBuilder<?>> channelCustomization) {
```

Comment:
Can we add a test that such a `channelCustomization` is actually invoked upon creation of the `AxonServerConnectionManager`?

Reply:
added testcase